MathCalc.ts

```diff
@@ -0,0 +1,28 @@
+//Union Types
+type AlphaNumbers= (number | string);
+
+let AddConcatNumStr:(x:AlphaNumbers,y:AlphaNumbers)=>AlphaNumbers;
+
+AddConcatNumStr=(x:AlphaNumbers,y:AlphaNumbers)=>
+{
+   let result:AlphaNumbers="";
+
+   if(typeof x=="string" && typeof y=="string")
```

Comment:
Use triple `===` when you know the type needs to match, too. We use almost entirely `===` instead of `==`

Reply:
yes..corrected the condition to "==="